apollo-router/src/plugins/authentication/tests.rs

```diff
@@ -2,6 +2,8 @@ use std::collections::HashMap;
 use std::collections::HashSet;
 use std::path::Path;
 
+use base64::prelude::BASE64_URL_SAFE_NO_PAD;
+use base64::Engine as _;
```

Comment:
As one thing I found odd, I found that it was required to `use` the `Engine`, even though it's not used directly.  The `BASE64_URL_SAFE_NO_PAD` didn't seem to compile without that.

Thoughts? Or is this kind of module side-effect expected/typical.

Reply:
This is typical when needing to use trait methods.